src/store/modules/nft.js

```diff
@@ -230,13 +231,19 @@ const actions = {
         nextKey = data.pagination.next_key;
         ({ count } = data.pagination);
         nfts.push(...data.nfts);
-      } while (nextKey);
+      } while (count === NFT_INDEXER_LIMIT_MAX);
       // sort by last colleted by default
       return nfts.map(formatNFTInfo).sort((a, b) => b.timestamp - a.timestamp);
```

Comment:
Is this sorting necessary? Seem should be controlled by getters

Reply:
If two NFTs get the same rank in the comparator (e.g. same price), the UI will display the query result's default order(`ORDER BY id` in the table). That will be a little arbitrary for users. So I sort the NFTs here to achieve the effect similar to `ORDER BY ${rule from getter} THEN BY timestamp DESC`. 
Should we keep this and add some comments here?